### ISSUE TITLE
Weird startup error message

### ISSUE THREAD

**Issue by JeromeLegras** [sdllc/Basic-Excel-R-Toolkit#48]
Hello,
I keep getting this weird startup error message on startup: 
"Sourced file OK: ExcelFunctions.R
Sourced file OK: functions.R
Error in file(filename, "r", encoding = encoding) : 
  cannot open the connection
In addition: Warning message:
In file(filename, "r", encoding = encoding) :
  cannot open file 'ExcelFunctions.R': No such file or directory
Error reading startup file @ Fri Apr 21 14:59:30 2017"

Of course, what's odd is the combination of "Sourced file OK: ExcelFunctions.R" & "cannot open file 'ExcelFunctions.R': No such file or directory"

I have tried to change the startup directory etc but with no result.
Any clues ? 
thanks !

**Comment by duncanwerner** [comment]
That's probably a stray include.  Did you upgrade from an older version?  We changed around the file structure at some point, but if you had an older file it probably left it in place.

Check if `functions.R` has a `source(ExcelFunctions.R)` in it, and if so, remove it.

**Comment by JeromeLegras** [comment]
Yes I upgraded. I had a source(ExcelFunctions.R), removed it, but still have the same error message...

**Comment by duncanwerner** [comment]
That is the problem, though.  You might not be editing the right file, or you might have two of those includes.  Startup files are now under `My Documents`, so have a look in there.  Try putting some `print()` statements into files to see when they're loaded.

**Comment by JeromeLegras** [comment]
thanks again ! so i deleted all the redundant ExcelFunctions.R & functions.R that had on the disk drive (including some in the AppData/Roaming etc directory) and now the error message has changed :  
Basic Excel R Toolkit (BERT) v 1.51.0
Copyright (C) 2014-2017 Structured Data, LLC

Sourced file OK: ExcelFunctions.R
File watch failed.  Make sure the files you are watching exist and are readable.
Sourced file OK: functions.R

The  function.R has the following code line though : 
BERT$WatchFile( file.path( BERT$HOME, "functions.R" ))

is this where a different directory should be used ? Because BERT$HOME is not the same directory as the startup folder...

**Comment by duncanwerner** [comment]
Just take that out.  Everything in the startup folder is now watched automatically.  You only need to explicitly watch if you want to do something other than load the file.
